### PR TITLE
Ulii

### PR DESCRIPTION
**Pull request by actlikewill**
- Setup Ulii for Peachjam

### CODE REVIEW

File: ulii/static/stylesheets/_variables.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+$primary: red;
```
Comment (longhotsummer) [inline ulii/static/stylesheets/_variables.scss:1]:
```suggestion
$primary: #2b2e90;
```

File: ulii/templates/liiweb/home.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  </section>
+
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Legislation' %}</h2>
+    {% trans 'View more legislation' as text %}
+    {% include 'peachjam/_recent_document_list.html' with documents=recent_legislation view_more_text=text view_more_url_name='legislation_list' %}
+  </section>
+
+  <section class="pb-3 pt-3">
+    <h2 class="mb-4 heading-underlined">{% trans 'Recent Articles' %}</h2>
```
Comment (longhotsummer) [inline ulii/templates/liiweb/home.html:20]: You can drop the articles block.

File: ulii/templates/peachjam/_header.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link {% if view.navbar_link == 'taxonomy' %}active{% endif %}" href="{% url 'top_level_taxonomy_list' %}">
+      {% trans 'Taxonomy' %}
+    </a>
+  </li>
+  <li class="nav-item">
+    <a class="nav-link {% if view.navbar_link == 'articles' %}active{% endif %}" href="{% url 'article_list' %}">
+      {% trans 'Articles' %}
```
Comment (longhotsummer) [inline ulii/templates/peachjam/_header.html:28]: Remove articles

File: ulii/views/home.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+class HomePageView(BaseHomePageView):
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["recent_judgments"] = Judgment.objects.order_by("-date")[:5]
+        context["recent_legislation"] = Legislation.objects.filter(
+            metadata_json__stub=False
+        ).order_by("-date")[:10]
+        context["recent_articles"] = (
```
Comment (longhotsummer) [inline ulii/views/home.py:13]: drop articles

File: ulii/static/stylesheets/_variables.scss (hunk 1 above)
Comment (actlikewill) [reply ulii/static/stylesheets/_variables.scss]: :heavy_check_mark:

File: ulii/templates/liiweb/home.html (hunk 2 above)
Comment (actlikewill) [reply ulii/templates/liiweb/home.html]: :heavy_check_mark:

File: ulii/templates/peachjam/_header.html (hunk 3 above)
Comment (actlikewill) [reply ulii/templates/peachjam/_header.html]: :heavy_check_mark:

File: ulii/views/home.py (hunk 4 above)
Comment (actlikewill) [reply ulii/views/home.py]: :heavy_check_mark:

**Review by longhotsummer — APPROVED** [review]
Great. Once we have the first deployment, we can setup automatic deployments.

**Pull request merged**